src/styles/utils/_keyframes.scss

```diff
@@ -0,0 +1,11 @@
+@keyframes move {
+  from {
+    transform: rotate(-90deg);
+    transform-origin: left;
```

Comment:
Set transform origin property to the general class, it doesn't change till animation